test/regtest.py

```diff
@@ -14,7 +19,7 @@
 
 def red(text):
   return '\033[0;31m' + text + '\033[0m'
-  
+
 def green(text):
   return '\033[0;32m' + text + '\033[0m'
 
```

Comment:
Are these even needed anymore. They're not being used by the logging merely because the logging doesn't have a clean solution to do colored out. At least from what I've been reading, and the examples I've seen..